Library/Formula/twoping.rb

```diff
@@ -0,0 +1,18 @@
+class Twoping < Formula
+  desc "Ping utility to determine directional packet loss"
+  homepage "http://www.finnie.org/software/2ping/"
+  url "http://www.finnie.org/software/2ping/2ping-3.0.1.tar.gz"
+  sha256 "d6997cd1680151e6f7d5e60137d45cd41bf385d26029878afdaaf5dc4f63dcc4"
+
+  depends_on "Python"
+
+  def install
+    system "python", *Language::Python.setup_install_args(prefix)
+    man1.install "doc/2ping.1"
+    man1.install_symlink "2ping.1" => "2ping6.1"
+  end
+
+  test do
+    assert_equal "#{version}", shell_output("#{bin}/2ping -V 2>&1").chomp
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!